examples/nirf/rfsa-rfsg-NR-analysis-only-modacc.py

```diff
@@ -0,0 +1,464 @@
+            )
+        )
+    )
+    raise_if_error(
+        rfsaclient.ConfigureIQCarrierFrequency(
+            nirfsa_types.ConfigureIQCarrierFrequencyRequest(
+                vi=rfsasession, carrier_frequency=center_frequency
+            )
+        )
+    )
+    ###### Configure RFSG Settigs ######
+    raise_if_error(
+        rfsgclient.SetAttributeViReal64(
+            nirfsg_types.SetAttributeViReal64Request(
+                vi=rfsgsession,
+                channel_name="",
+                attribute_id=nirfsg_types.NIRFSG_ATTRIBUTE_POWER_LEVEL,
+                value_raw=power_level_dbm,
+            )
+        )
+    )
+    raise_if_error(
+        rfsgclient.SetAttributeViReal64(
+            nirfsg_types.SetAttributeViReal64Request(
+                vi=rfsgsession,
+                channel_name="",
+                attribute_id=nirfsg_types.NIRFSG_ATTRIBUTE_FREQUENCY,
+                value_raw=center_frequency,
+            )
+        )
+    )
+    raise_if_error(
+        rfsgclient.ConfigureGenerationMode(
+            nirfsg_types.ConfigureGenerationModeRequest(
+                vi=rfsgsession, generation_mode=nirfsg_types.GENERATION_MODE_SCRIPT
+            )
+        )
+    )
+    raise_if_error(
+        rfsgclient.ReadAndDownloadWaveformFromFileTDMS(
+            nirfsg_types.ReadAndDownloadWaveformFromFileTDMSRequest(
+                vi=rfsgsession,
+                waveform_name=rfsg_waveform_name,
+                file_path=str(file_path),
+                waveform_index=0,
+            )
+        )
+    )
+    raise_if_error(
+        rfsgclient.WriteScript(nirfsg_types.WriteScriptRequest(vi=rfsgsession, script=rfsg_script))
+    )
+    raise_if_error(
+        rfsgclient.ExportSignal(
+            nirfsg_types.ExportSignalRequest(
+                vi=rfsgsession,
+                signal=nirfsg_types.ROUTED_SIGNAL_MARKER_EVENT,
+                signal_identifier_mapped=nirfsg_types.SIGNAL_IDENTIFIER_MARKER0,
+                output_terminal_mapped=nirfsg_types.OUTPUT_SIGNAL_PXI_TRIG1,
+            )
+        )
+    )
+
+    ###### Configure NR Settigs ######
+    nr_frequency_range = nirfmxnr_types.NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE1
+    nr_carrier_bandwidth = 100e6
+    nr_subcarrier_spacing = 30e3
+    nr_auto_resource_block_detection_enabled = True
+    nr_pusch_modulation_type = nirfmxnr_types.NIRFMXNR_INT32_PUSCH_MODULATION_TYPE_QAM64
+    nr_measurement_length_unit = nirfmxnr_types.NIRFMXNR_INT32_MODACC_MEASUREMENT_LENGTH_UNIT_SLOT
+    nr_link_direction = nirfmxnr_types.NIRFMXNR_INT32_LINK_DIRECTION_UPLINK
+    nr_measurement_offset = 0
+    nr_measurement_length = 1
+    raise_if_error(
+        rfmxclient.SetAttributeI32(
+            nirfmxnr_types.SetAttributeI32Request(
+                instrument=rfmxsession,
+                selector_string="",
+                attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_LINK_DIRECTION,
+                attr_val_raw=nr_link_direction,

```

Comment:
More curious than anything, but is there a reason for this and the next one why you're using `attr_val_raw` instead of just `attr_val`?

Reply:
You are correct. I copied the attribute method and didn't catch it. I updated it to attr_val